### PR TITLE
Removes lean

### PR DESCRIPTION
**Pull request by Thunder12345**
## About The Pull Request

Removes lean, its sprite, and all appearances of it in code.

## Why It's Good For The Game

Lean is an annoying, overused flavour of the month meme, totally devoid of character or originality. The only reason for it to exist here is because somebody saw it once, giggled for a few milliseconds, and made a maintainer giggle for a similar length of time.

It serves no useful gameplay purpose, and which I have never personally seen even being used, yet its mere existence empowers and legitimises people to reach sometimes borderline OOC in IC levels of talking about lean, safely protected by the shield of lean existing as an ingame thing regardless of how rarely seen.

Due to this ability to cover for annoying behaviour, its rarity and its lack of functional purpose in gameplay when actually made, I do not see any way in which lean existing is not a net negative.

tl;dr lean sucks

## Changelog

:cl:
del: Lean has been removed entirely from the game
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
"the month"

> added 2020 https://github.com/tgstation/tgstation/pull/50029

**Comment by SmArtKar** [comment]
Remove all the other drinks then too because most of them "don't have a purpose" either

**Comment by SweptWasTaken** [comment]
> yet its mere existence empowers and legitimises people to reach sometimes borderline OOC in IC levels of talking about lean.

So when are we removing meth and weed?

**Comment by Thunder12345** [comment]
> "the month"
> 
> > added 2020 #50029
Flavour of the month memes do not necessarily start their month the moment they're added, sometimes it takes a while for the most annoying of the community to latch onto something and make it unbearable


> Remove all the other drinks then too because most of them "don't have a purpose" either
The other drinks don't have an annoying as fuck subculture latching onto them and inflicting themselves on everyone in earshot

**Comment by Jolly-66** [comment]
But I love lean..

**Comment by rellyreri** [comment]
how do you get robusted by lean

**Comment by datorangebottle** [comment]
Thank god.

**Comment by SweptWasTaken** [comment]
> mrp has an annoying subculture attached to it when can remove it

Holy based

**Comment by Colovorat** [comment]
> mrp has an annoying subculture attached to it when can remove it

Despite everything, he has a point

**Comment by bruhlookatthisdood** [comment]
BRO NO
MY LEAN FACTORY
MY LEAN!!!!!!

**Comment by nianjiilical** [comment]
are you telling me lean has actually been in the game this entire time and ive been wasting the effort put into spawning purple-tinted pruno bags filled with every negative chemical in existence

**Comment by TheVekter** [comment]
Y'all, if we removed everything that had an annoying meme around it, we be removing everything.

This is an administration issue, not a code issue.

**Comment by RiskySikh** [comment]
Hi I am the creator of this Lean PR, I made this PR almost 2 years ago and at the time it wasn't a meme. For me I barely code, and me and twaticus put a lot of time and effort into this creation. I want to see my contributions live on in /tg/, even though I am inactive and barely play I am happy to see my creations, and it will be sad if they are gone. I agree that the lean meme is overused and beaten, but I made this PR before this trend so there is no use of removing it.

**Comment by RiskySikh** [comment]
You know whats even more ironic in 2 more days its gonna be the 2 year anniversary of my learn PR

**Comment by optimumtact** [comment]
remove the stick from your ass please

**Comment by optimumtact** [comment]
also can someone tell me what this was really about?

**Comment by LemonInTheDark** [comment]
> 
> its mere existence empowers and legitimises people to reach sometimes borderline OOC in IC levels of talking about lean, safely protected by the shield of lean existing as an ingame thing regardless of how rarely seen.

I don't like lean either, but would you please fuck off with this?
It is not our job to manage players being idiots for you, I really hate seeing this kind of pr because it's like...
Someone does something absolutely moronic relating to lean -> Admin wants to tell them to stop being dumb, is afraid to because it's technically in code -> Makes pr removing it.

This happen with erp posters too and it drove me absolutely mental. We're not actively trying to make your job worse, but just fucking ban them. We can't just remove everything some dumbass on the servers tries to use as a defense for something cringe they're doing.

**Comment by optimumtact** [comment]
creator of this drink is a fucking moron

**Comment by LemonInTheDark** [comment]
it has begun

**Comment by ShibaInuLord** [comment]
we do a little trolling

**Comment by DogZeroX** [comment]
Based.

**Comment by DogZeroX** [comment]
unknown

**Comment by Shadowflame909** [comment]
Grape soda tastes better anyways

**Comment by Sk1Wass3r** [comment]
Now lean got 1984'd from both tg and 14... zamn

**Comment by theselfish** [comment]
Mug Maniacs stay winning.

**Comment by DAKKA-WAAAGH** [comment]
One of the worst PRs ever to exist.

**Comment by bruhlookatthisdood** [comment]
can agree
